dynamodb-quickstart/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
```

Comment:
Don't include the default landing page in quickstarts

Reply:
removed